src/main/kotlin/me/glaremasters/guilds/actions/ActionHandler.kt

```diff
@@ -21,29 +21,27 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+package me.glaremasters.guilds.actions
 
-package me.glaremasters.guilds.api.events;
+import org.bukkit.command.CommandSender
+import java.util.*
 
-import me.glaremasters.guilds.api.events.base.GuildEvent;
-import me.glaremasters.guilds.guild.Guild;
-import org.bukkit.entity.Player;
+class ActionHandler {
+    private val actions: MutableMap<CommandSender, ConfirmAction> = HashMap()
```

Comment:
You should almost never be using the actual collection type constructors, the stdlin ships with (way too) many functions for creating collections.

```kotlin
private val actions = mutableMapOf<CommandSender, ConfirmAction>()
```